README.md

```diff
@@ -190,6 +183,26 @@ gem 'github-linguist'
 
 _Note:_ See [Gemfile](/Gemfile) `:test` block for version requirements.
 
+
+For example, `SyntaxHighlightFilter` uses [github-linguist](https://github.com/github/linguist)
+to detect and highlight languages. To use the `SyntaxHighlightFilter`, add the following to your Gemfile:
+
+```ruby
+gem 'github-linguist'
+```
+
+In turn, the actual syntax highlighting is handled by [pygments.rb](https://github.com/tmm1/pygments.rb). `Pygments.rb` can generate the styles.  The following generates the CSS for HTML formatted code:
+
+```
+❯ irb
```

Comment:
Could we use `$` to indicate its a command? Follow [the conventions](https://github.com/jch/html-pipeline#installation).